src/views/protected/MyAccount/MyAccountPage.tsx

```diff
@@ -0,0 +1,128 @@
+    const [password, setPassword] = useState('')
+    const [confirmPassword, setConfirmPassword] = useState('')
+    return <div className={classes.root}>
+        <FirebaseContext.Consumer>
+            {(firebase: Firebase): React.ReactNode => <AppBar firebase={firebase} />}
+        </FirebaseContext.Consumer>
+        <div className={classes.formContainer}>
+            <Grid container
+                  direction="column"
+                  justify="flex-start"
+                  alignItems="center"
+                  style={{ width: '100vw', height: '100%', paddingTop: '1em' }}>
+                <Grid item xs={9}>
+                    <Typography style={{ fontSize: '2.5em' }}>
+                        My Account
+                    </Typography>
+                </Grid>
+                <Grid item xs={8} spacing={8} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <InputLabel id="demo-mutiple-name-label">Name</InputLabel>
+                        <Input onChange={(event: React.ChangeEvent<HTMLInputElement>) => setName(event.target.value)}
+                               value={name} />
+                    </StyledFormControl>
+                </Grid>
+                <Grid item xs={8} spacing={8} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <InputLabel id="demo-mutiple-name-label">Email</InputLabel>
+                        <Input onChange={(event: React.ChangeEvent<HTMLInputElement>) => setEmail(event.target.value)}
+                               value={email} />
+                    </StyledFormControl>
+                </Grid>
+                <Grid item xs={8} spacing={2} className={classes.container}>
+                    <StyledFormControl className={classes.formControl}>
+                        <FormHelperText>Enter a new password twice to change your password</FormHelperText>
+                        <InputLabel id="demo-mutiple-name-label">Password</InputLabel>
+                        <Input type="password" placeholder="Password"
+                               onChange={(event: React.ChangeEvent<HTMLInputElement>) => setPassword(event.target.value)}
+                               value={password} />
+                        <Input type="password" placeholder="Confirm"
+                               onChange={(event: React.ChangeEvent<HTMLInputElement>) => setConfirmPassword(event.target.value)}
+                               value={confirmPassword} />
+                    </StyledFormControl>
+                </Grid>
+                <Grid item xs={8} spacing={2} className={classes.container}>
+                    <FirebaseContext.Consumer>
+                        {(firebase:Firebase) => (
+                            <Button variant="contained" color="primary"
+                                    onClick={(_) =>
```

Comment:
@thompchr I think they wanted some sort of confirmation pop-up when the user clicks 'save'